packages/react-instantsearch-hooks-server/src/__tests__/modules-none.test.tsx

```diff
@@ -114,9 +114,25 @@ describe('ReactDOMServer imports', () => {
     await expect(
       getServerState(<App />)
     ).rejects.toThrowErrorMatchingInlineSnapshot(
-      `"Could not import ReactDOMServer."`
+      `"Could not import ReactDOMServer. You can provide it as an argument: getServerState(<Search />, { renderToString })."`
     );
   });
+
+  test('does not throw if user provides their own renderToString', async () => {
```

Comment:
It better describes what the test does:

```suggestion
  test('calls the provided renderToString function', async () => {
```